src/vs/workbench/services/suggest/browser/simpleSuggestWidget.ts

```diff
@@ -227,9 +227,10 @@ export class SimpleSuggestWidget extends Disposable {
 		this._details = this._register(new SimpleSuggestDetailsOverlay(details, this._listElement));
 		this._register(dom.addDisposableListener(this._details.widget.domNode, 'blur', (e) => this._onDidBlurDetails.fire(e)));
 
-		if (options.statusBarMenuId) {
-			this._status = this._register(instantiationService.createInstance(SuggestWidgetStatus, this.element.domNode, options.statusBarMenuId));
+		if (_options.statusBarMenuId) {
```

Comment:
We'll want a setting to drive this for the terminal as well, ours will just be enabled by default

Reply:
Yep, agreed